firebase_rules_generator/BUILD

```diff
@@ -37,5 +38,6 @@ cc_binary(
 
 cc_proto_library(
     name = "firebase_rules_options",
+    visibility = ["//visibility:public"],
```

Comment:
Do this libraries need to be public? If anything this should be restricted to this project only.

Reply:
This is part of my workaround for https://github.com/FirebaseExtended/protobuf-rules-gen/issues/25

Reply:
Can you instead do: `package(default_visibility = ["//visibility:public"])`